pkg/rbac/userData_test.go

```diff
@@ -394,7 +399,8 @@ func Test_managedClusters_emptyCache(t *testing.T) {
 		return true, notCreateRule, nil
 
 	})
-	result, err := mock_cache.GetUserData(context.Background(), "123456", fs.AuthorizationV1())
+	ctx := context.WithValue(context.TODO(), ContextAuthTokenKey, "123456")
```

Comment:
Actually `context.Background()` is better for the tests.  It yields the same and it doesn't imply that we should change it later.

Reply:
Updated